6-Typescript/server/tests/todo.test.ts

```diff
@@ -0,0 +1,171 @@
+
+import { AxiosError } from 'axios'
+import { Testkit } from './env/testkit'
+import { Item } from '../interfaces/Item'
+import { AppDriver } from './drivers/appDriver'
+import { DBDriver } from './drivers/DBDriver'
+import { MongoDBController } from '../db/mongoDBController'
+import {v4 as uuid} from 'uuid'
+
+const db = new MongoDBController()
+const appDriver: AppDriver = new AppDriver('http://localhost:3000/')
+const dbDriver: DBDriver = new DBDriver(db)
+const testkit: Testkit = new Testkit(appDriver, dbDriver)
+const existingItemID: string = uuid()
+const nonExistingItemID: string = uuid()
+
+describe("testing the app's endpoints'", () => {
+    testkit.app.start(db)
+    test("get all items from a user", async () => {
+        await testkit.before()
```

Comment:
this is meant to be called before each test right?
so the best place for it is in the `beforeEach` block in the describe.
plz read about jest beforeEach / before / etc.